src/System.Diagnostics.EventLog/src/System/Diagnostics/Reader/EventPropertyContext.cs

```diff
@@ -0,0 +1,28 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+/*============================================================
+**
+**
+** Purpose:
+** This public abstract class defines the methods / properties
+** for a context object used to access a set of Data Values from
+** an EventRecord.
+**
+============================================================*/
```

Comment:
remove this

Reply:
I suggest retaining any useful text but use `<summary>`